src/SignalR/common/Http.Connections/src/ConnectionOptions.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace Microsoft.AspNetCore.Http.Connections
 {
+    /// <summary>
+    /// Options used to change behavior of how connections are handled when using HTTP.
```

Comment:
Right now this is only used by long polling, right? Is there any reason we wouldn't use this for non-HTTP based transports in the future?

Reply:
This is used for all HTTP connections right now in the scan method, no reason it can't be used for non-HTTP